src/js/popup.ts

```diff
@@ -221,7 +228,8 @@ function setUpBackgroundMessageHandler(tabId: string | null): void {
 (function (): void {
   const params = new URL(document.location.href).searchParams;
   setUpBackgroundMessageHandler(params.get('tab_id'));
-  updateDisplay(params.get('state'));
+  const state = params.get('state');
+  state && updateDisplay(state);
```

Comment:
Would we want to update display to invalid here if we have a missing state param? Or would that be too draconian?

Reply:
I feel it's a bit unnecessary 

We set the params here https://github.com/facebookincubator/meta-code-verify/blob/main/src/js/background/tab_state_tracker/TabStateMachine.ts#L88 but we can't really strictcly type this, but it's simple enough for manual verification in this case.